lib/participant.js

```diff
@@ -503,6 +504,7 @@ class Participant extends EventEmitter {
  * A {@link Track} was removed by the {@link Participant}.
  * @param {Track} track - The {@link Track} that was removed
  * @event Participant#trackRemoved
+ * @deprecated
```

Comment:
For RemoteParticipants, we can suggest "trackUnsubscribed". For LocalParticipants, I don't believe we'll be emitting an event anymore?

Reply:
@markandrus In `LocalParticipant`, "trackAdded" will be used internally.

Reply:
Yes, but our `@deprecated` warning here applies to the super-class. So for our customers using the public API, we should explain what they should do in the LocalParticipant case.